Install/install.sh

```diff
@@ -16,11 +16,20 @@ SERVICE_FILE_TEMPLATE=slowmovie.service.template
 
 function install_linux_packages(){
   sudo apt-get update
-  sudo apt-get install -y ffmpeg git python3-pip libatlas-base-dev
+  sudo apt-get install -y ffmpeg git python3-pip python3-venv libatlas-base-dev
+}
+
+function create_python_venv(){
+  python3 -m venv --system-site-packages $LOCAL_DIR/.venv # numpty requires system-site-packages
+  source $LOCAL_DIR/.venv/bin/activate #activate our venv
 }
 
 function install_python_packages(){
-  pip3 install -r $LOCAL_DIR/Install/requirements.txt -U
+  # fix PEP 517 error with OmniEPD
+  pip3 install --upgrade pip setuptools wheel
+  #fix for IT8951 install https://github.com/GregDMeyer/IT8951/issues/58
```

Comment:
This shouldn't be needed anymore with the latest IT8951 repo fixes. I tested building on both Bullseye and Bookworm and both installed without error now

Reply:
Agreed. I just tested it locally and with the upstream changes to IT8951 those workarounds aren't necessary anymore.